nectar/downloaders/local.py

```diff
@@ -211,7 +212,7 @@ def _common_link(self, link_method, request, report=None):
             report.bytes_downloaded = os.path.getsize(request.destination)
 
         except OSError, e:
-            logger.debug(e)
+            logger.info(e)
```

Comment:
From my testing, this usually displays the path (e.g. `[Errno 13] Permission denied: '///home/vagrant/zoolander/cat-1.0-1.noarch.rpm'`)